js/anims-missiles.js

```diff
@@ -0,0 +1,94 @@
+//global scope
+var columnID = 0
+var gameScore = {
+    score:0,
+    toll:0
+}
+
+function dropMissile(animationTime) {
+    var columnNumber = randomNumGen(4)
+    var $missile = $('<img src="img/missile-drop-down-white.png" ' +
+        'class="missile missile-'+ columnNumber +'">')
+    $('#pos-'+columnNumber).prepend($missile)
+
+    $missile.animate({
+        top: "+=550"
+    }, animationTime, "linear", function () {
+        missileHitsCity()
+        $(this).remove()
+    })
+}
+
+function repeatAnims() {
+    setInterval(function() {
+        dropMissile(2000)
+    }, 500)
+}
+
+function missileHitsCity() {
+    gameScore.score -= 1
+    gameScore.toll += 10000
+    document.querySelector('#score').textContent = gameScore.score
+    document.querySelector('#toll').textContent = gameScore.toll
+}
+
+function randomNumGen(topLimit) {
+    var randomNum = Math.ceil(Math.random() * topLimit)
+    return randomNum
```

Comment:
just return the line above, no need to create the variable, wasting JS memory

Reply:
Fixed in fc27f2a